src/openrct2/Game.h

```diff
@@ -16,90 +16,94 @@
-    GAME_COMMAND_LOWER_WATER,                  // GA
-    GAME_COMMAND_SET_BRAKES_SPEED,             // GA
-    GAME_COMMAND_HIRE_NEW_STAFF_MEMBER,        // GA
-    GAME_COMMAND_SET_STAFF_PATROL,             // GA
-    GAME_COMMAND_FIRE_STAFF_MEMBER,            // GA
-    GAME_COMMAND_SET_STAFF_ORDERS,             // GA
-    GAME_COMMAND_SET_PARK_NAME,                // GA
-    GAME_COMMAND_SET_PARK_OPEN,                // GA
-    GAME_COMMAND_BUY_LAND_RIGHTS,              // GA
-    GAME_COMMAND_PLACE_PARK_ENTRANCE,          // GA
-    GAME_COMMAND_REMOVE_PARK_ENTRANCE,         // GA
-    GAME_COMMAND_SET_MAZE_TRACK,               // GA
-    GAME_COMMAND_SET_PARK_ENTRANCE_FEE,        // GA
-    GAME_COMMAND_SET_STAFF_COLOUR,             // GA
-    GAME_COMMAND_PLACE_WALL,                   // GA
-    GAME_COMMAND_REMOVE_WALL,                  // GA
-    GAME_COMMAND_PLACE_LARGE_SCENERY,          // GA
-    GAME_COMMAND_REMOVE_LARGE_SCENERY,         // GA
-    GAME_COMMAND_SET_CURRENT_LOAN,             // GA
-    GAME_COMMAND_SET_RESEARCH_FUNDING,         // GA
-    GAME_COMMAND_PLACE_TRACK_DESIGN,           // GA
-    GAME_COMMAND_START_MARKETING_CAMPAIGN,     // GA
-    GAME_COMMAND_PLACE_MAZE_DESIGN,            // GA
-    GAME_COMMAND_PLACE_BANNER,                 // GA
-    GAME_COMMAND_REMOVE_BANNER,                // GA
-    GAME_COMMAND_SET_SCENERY_COLOUR,           // GA
-    GAME_COMMAND_SET_WALL_COLOUR,              // GA
-    GAME_COMMAND_SET_LARGE_SCENERY_COLOUR,     // GA
-    GAME_COMMAND_SET_BANNER_COLOUR,            // GA
-    GAME_COMMAND_SET_LAND_OWNERSHIP,           // GA
-    GAME_COMMAND_CLEAR_SCENERY,                // GA
-    GAME_COMMAND_SET_BANNER_NAME,              // GA
-    GAME_COMMAND_SET_SIGN_NAME,                // GA
-    GAME_COMMAND_SET_BANNER_STYLE,             // GA
-    GAME_COMMAND_SET_SIGN_STYLE,               // GA
-    GAME_COMMAND_SET_PLAYER_GROUP,             // GA
-    GAME_COMMAND_MODIFY_GROUPS,                // GA
-    GAME_COMMAND_KICK_PLAYER,                  // GA
-    GAME_COMMAND_CHEAT,                        // GA
-    GAME_COMMAND_PICKUP_GUEST,                 // GA
-    GAME_COMMAND_PICKUP_STAFF,                 // GA
-    GAME_COMMAND_BALLOON_PRESS,                // GA
-    GAME_COMMAND_MODIFY_TILE,                  // GA
-    GAME_COMMAND_EDIT_SCENARIO_OPTIONS,        // GA
-    GAME_COMMAND_PLACE_PEEP_SPAWN,             // GA
-    GAME_COMMAND_SET_CLIMATE,                  // GA
-    GAME_COMMAND_SET_COLOUR_SCHEME,            // GA
-    GAME_COMMAND_SET_STAFF_COSTUME,            // GA
-    GAME_COMMAND_PLACE_FOOTPATH_SCENERY,       // GA
-    GAME_COMMAND_REMOVE_FOOTPATH_SCENERY,      // GA
-    GAME_COMMAND_GUEST_SET_FLAGS,              // GA
-    GAME_COMMAND_SET_DATE,                     // GA
-    GAME_COMMAND_CUSTOM,                       // GA
-    GAME_COMMAND_COUNT,
+    PasswordlessLogin = -3,
+    ToggleSceneryCluster = -2,
+    Chat = -1,
```

Comment:
I'm not yet convinced this is correctly working (`::Count` probably doesn't include the negatives).

Reply:
No that is correct. Its stupid and it should be changed but Count needs to ignore those special ones for now